parameter/SelectionCriterionType.cpp

```diff
@@ -51,6 +53,18 @@ std::string CSelectionCriterionType::getKind() const
 // From ISelectionCriterionTypeInterface
 bool CSelectionCriterionType::addValuePair(int iValue, const std::string& strValue)
 {
+    // An inclusive criterion is implemented as a bitfield over an int and
+    // thus, can't have values larger than the number of bits in an int.
```

Comment:
This comment is not correct. How about:
~~~ cpp
 // Inclusive criterion is implemented as a bitfield over an int.
 // As such fitfield are usually constructed by shifting a bit, forbid the sign bit to be used.
 // Shifting a bit over the sign bit returns in an overflow, which is an undefined behavior.
~~~